docs/sqlalchemy.txt

```diff
@@ -399,3 +400,59 @@ delete a single table use ``drop(...)`` as shown below::
 
     >>> engine.dialect.has_table(conn, table_name='departments')
     False
+
+
+Insert From Select
+==================
+
+In SQLAlchemy, the `insert().from_select()` function returns a new `Insert`
```

Comment:
single backticks (italics) should be replaced with double backticks (monospaced font)